client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -150,6 +158,25 @@ const onboarding: Flow = {
 
 		const [ , isGoalsAtFrontExperiment ] = useGoalsFirstExperiment();
 
+		const { isEligible: isBigSkyEligible } = useIsBigSkyEligible();
+		const isDesignChoicesStepEnabled = isBigSkyEligible && isGoalsAtFrontExperiment;
+
+		const getPostCheckoutDestination = ( providedDependencies: ProvidedDependencies ) => {
+			if ( createWithBigSky ) {
+				return addQueryArgs( '/setup/site-setup/launch-big-sky', {
```

Comment:
We're getting into Big Sky by jumping into a very specific step of the `site-setup` flow. It's kinda weird but seems to work. I wouldn't be surprised if we need to modify this logic later.